presidio-anonymizer/presidio_anonymizer/operators/custom.py

```diff
@@ -0,0 +1,44 @@
+"""Replaces the PII text with lambda result."""
+from typing import Dict
+
+from presidio_anonymizer.operators import Operator, OperatorType
+from presidio_anonymizer.services.validators import validate_type
+from presidio_anonymizer.entities import InvalidParamException
+
+
+class Custom(Operator):

```

Comment:
nit: are we sure about the name? why not call this lambda?

Reply:
name still WIP, open for suggestions :) isn't lambda a reserved name?

Reply:
lambda yes, Lambda no :) we can put Lambda as a class it won't conflict. But need to think about it.

Reply:
not sure about the name, discuss offline